lib/alephant/renderer/views.rb

```diff
@@ -8,13 +8,13 @@ def self.register(klass)
       end
 
       def self.get_registered_class(id)
-        @@views[id]
+        @@views[id.downcase]
```

Comment:
@Integralist, I also added this to get the models in the tests to match, as I noticed when using uppercase letters, they didn't match, so the regex for uppercases wouldn't have worked.

I had a choice of this or `underscorify(id)`. Let me know if you prefer that one, or have another suggestion.